### PR TITLE
Activity post form Gif icon highlighted issue fixed

### LINKED ISSUE #1197: Minor formatting issue with "gif" and "Emoticons" icon on the activity page
**Describe the bug**
Minor formatting issue with "gif" and "Emoticons" icon on the activity page. When we hover over the gif icon, it displays all the gifs but when we remove the cursor, the preview still stays. Please watch the video to understand the issue. 

**To Reproduce**
1. Go to the activity page
2. Hover over the gif icon
3. See error

**Expected behavior**
The preview should not display if we remove the cursor from the gif icon.

**Screenshots**
https://www.youtube.com/watch?v=ZO1Mxw4HCBY&feature=youtu.be

**Support ticket links**
https://secure.helpscout.net/conversation/1224172627/83133/

### PR DESCRIPTION
**Pull request by mahedihasannoman**
### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes #1197  .

### How to test the changes in this Pull Request:

1. Go to the activity page
2. Click on the Gif icon
3. Click outside anywhere
4. See the error is not happening

### Proof Screenshots or Video

https://www.loom.com/share/124309e68c0147a7b0419789ca00b753

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

File: src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js [hunk 1]
```diff
@@ -2752,6 +2752,15 @@ window.bp = window.bp || {};
...
 
+				if ( ! _.isUndefined( BP_Nouveau.media ) && ! _.isUndefined( BP_Nouveau.media.gif_api_key ) ) {
+
+					if( $targetEl.parent('#activity-gif-button').length == 0 && ! $targetEl.parent('.gif-search-query').length && ! $targetEl.parent('.gif-media-search-dropdown').length && ! $targetEl.parent('.gif-media-search').length && ! $targetEl.parent('.gif-search-content').length ) {
+						if ( $( '#activity-gif-button' ).hasClass( 'open' ) ) {
+							$( '#activity-gif-button' ).removeClass( 'open' );
+						}
+					}
+				}
+
```
Comment (KartikSuthar) [inline src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js:2763]: @mahedihasannoman I don't think this is required Please remove it.

L2750
write else here 
```
					if ( gif_box.length && $.trim( gif_box.html() ) !== '' ) {
						this.$self.addClass( 'open' );
					} else {
						this.$self.removeClass( 'open' );
					}
					```

**Review by KartikSuthar — CHANGES_REQUESTED** [review]

File: src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js (hunk 1 above)
Comment (mahedihasannoman) [reply src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js:2763]: Ok, I am working on this.

**Review by KartikSuthar — DISMISSED** [review]

**Comment by JetalDobariya** [comment]
Hello ,

I did verify this PR's issue in QA-Ongoing branch, now Gif icon is not stay highlighted after click on out side of the Gif box.
So issue has been done and working fine as expected.

See Proof : https://www.loom.com/share/2eac1ef70d40401f858de688d347cc1b

Thanks!

**Comment by JetalDobariya** [comment]
Hello,

I did verify this PR in qa-approved branch, now GIF icon is not highlighted after click on out side of the GIF search box.

Thanks!

**Review by dipskakadiya — APPROVED** [review]

**Pull request merged**